plugins/time-date/PageComponent.qml

```diff
@@ -132,6 +139,70 @@ ItemPage {
                     })
                 }
             }
+            
+            SettingsItemTitle {
+                text: i18n.tr("In the top panel:")
+            }
+            
+            SettingsListItems.Icon {
+                id: calendarListItem
+                text: i18n.tr("Calendar")
+                iconName: "calendar"
+
+                Loader {
+                    sourceComponent: Switch {
+                        id: calendarSwitch
+                        property bool serverChecked: indicatorDatetime.showCalendar
+
+                        USC.ServerPropertySynchroniser {
+                            userTarget: calendarSwitch
+                            userProperty: "checked"
+                            serverTarget: calendarSwitch
+                            serverProperty: "serverChecked"
+
+                            onSyncTriggered: indicatorDatetime.showCalendar = checked
+                        }
+                    }
+                }
+            }
+            
+            SettingsListItems.Standard {
+                visible: indicatorDatetime.showCalendar
+                anchors.left: parent.left
+                anchors.leftMargin: units.gu(2)
+                CheckBox {
+                    objectName: "showWeekNumbers"
+                    SlotsLayout.position: SlotsLayout.First
+                    property bool serverChecked: indicatorDatetime.showWeekNumbers
+                    onServerCheckedChanged: checked = serverChecked
+                    Component.onCompleted: checked = serverChecked
+                    onTriggered: indicatorDatetime.showWeekNumbers = checked
+                }
+                text: i18n.tr("Show week numbers")
+            }
+            
+            /* Hidden for now because it stopped working sometime in the past
+            SettingsListItems.Icon {
+                id: eventsListItem
```

Comment:
I don't think we should commit new commented code. If it doesn't work, there's no need for it to be in the tree. By the time someone fixes it, this page might have been redesigned and the code won't work then either.